2021 03 02 iswilli accessreviewscopeupdates (#11842)
microsoftgraph/microsoft-graph-docs: api-reference/beta/resources/serviceprincipalidentity.md

---
title: "servicePrincipalIdentity resource type"

> description: "Models a service principal identity"

Reviewer: ```suggestion
description: "Models a service principal identity."
```

author: "isabelleatmsft"
localization_priority: Normal
ms.prod: "governance"
doc_type: resourcePageType
---

# servicePrincipalIdentity resource type

Namespace: microsoft.graph

[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]

> Models a service principal identity

Reviewer: ```suggestion
Models a service principal identity.
```

> Models a service principal identity

Reviewer: This doesn't really describe what a service principal identity is.

Inherits from [identity](../resources/identity.md).

## Properties
|Property|Type|Description|
|:---|:---|:---|

> |appId|String|app ID of the service principal|

Reviewer: ```suggestion
|appId|String| The application identifier of the service principal. |
```

> |displayName|String|Inherited from [identity](../resources/identity.md)|

Reviewer: ```suggestion
|displayName|String| The display name of the service principal identity. Inherited from [identity](../resources/identity.md)|
```
